onos-ric/values.yaml

```diff
@@ -27,14 +27,14 @@ store:
     partitions: 1
     clusters: 1
     backend:
-      image: atomix/local-replica:latest
+      image: atomix/local-replica:v0.1.0
```

Comment:
Needs to be rebased since these images changed. After that I think this PR is good